frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -200,6 +207,25 @@ const GraphicalSelector = () => {
     }
   };
 
+  function getZoomRatio() {
```

Comment:
prefer to use arrow function here

`const getZoomRatio = () => { ... } `

Reply:
yep! should be fixed with the update :D